docs/tutorials/object-management-kubectl/object-management.md

```diff
@@ -71,12 +71,19 @@ object in either YAML or JSON.
 The user provides an operation (create, replace, delete), one or more files,
 and flags to the `kubectl` command.
 
-This technique requires a deep understanding of the Kubernetes
-object definitions.
+This technique requires understanding the Kubernetes
+object definitions to create and update them.
```

Comment:
Feels like the first 4 paras (including the *Note*) reduce to:

In imperative object configuration,  the `kubectl` command specifies the operation (`create`, `replace`, etc.), optional flags and at least one file name. The file specified must contain a full definition of the object in YAML or JSON format.

followed by the Warning.  but up to you.  just stumbling over the word technique.